spec.html

```diff
@@ -16175,16 +16175,16 @@
               for (let key of Reflect.ownKeys(obj)) {
```

Comment:
Shouldn’t we use `const` over `let` throughout this code sample? None of the `let`s are reassigned.